data/companies/rea_group/corporate.yaml

```diff
@@ -0,0 +1,43 @@
+---
+type: corporate
+name: REA Group Ltd
+template: publisher
+facts:
+  - number_of_employees: 3145
+    source_id: 1
+    reference: page 29
+  - office_emissions_mt_co2e_per_employee_per_month: 0.047
+    calculation: true
+		reference: page 57
+  - commuting_emissions_mt_co2e_per_employee_per_month: 0.039
+    calculation: true
+		reference: page 57
+  - travel_emissions_mt_co2e_per_employee_per_month: 0.0490
+    calculation: true
+		reference: page 57
+  - datacenter_emissions_mt_co2e_per_employee_per_month: 0.0144
```

Comment:
This seems like it omitting scope 3 emissions from the software providers and other vendors that they use. I think we should omit.